docs/geo-shapes.rst

```diff
@@ -0,0 +1,55 @@
+==========================
+JanusGraph Geo Shapes
+==========================
+
+JanusGraph provides API to create Geometric objects so that they can
+be queried using Geo Predicates from `docs
+<geo-predicates.html>`_.
+
+
+----------------------------
+Available Geo Shapes
+----------------------------
+
+The current library has only following shapes implemented:
+
+    - POINT_
+    - CIRCLE_
+
+While JanusGraph's JVM based clients provides following shapes,
+Contributions are invited to implement the following shapes:
```

Comment:
Well, I'm okay with removing, but I just wanted to point out, which all geo shapes are implemented, and clear to other developers, so that if they want to extend, what all things they will need to implement extra. If i drop it here, where do I pass on this information?